From 85c44d01989df718f170abedbb942005f8c70f69 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Fri, 13 Feb 2015 00:59:04 +0100
Subject: [PATCH 33/56] updater: disable auto-update

As per http://www.chromium.org/administrators/turning-off-auto-updates ,
the auto update function is decidedly disabled on Linux, i.e.
the following patch is for Windows and MacOS.

For Windows, all we need is to build without -DGOOGLE_CHROME_BUILD (cf.
chrome/installer/util/google_update_settings.cc), which may already be
the case anyway, since we are based off Chromium, not Chrome.
---
 chrome/browser/app_controller_mac.mm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/app_controller_mac.mm b/chrome/browser/app_controller_mac.mm
index 137a060..96d8696 100644
--- a/chrome/browser/app_controller_mac.mm
+++ b/chrome/browser/app_controller_mac.mm
@@ -669,7 +669,7 @@ class AppControllerProfileObserver : public ProfileInfoCacheObserver {
   CFStringRef checkInterval = CFSTR("checkInterval");
   CFPropertyListRef plist = CFPreferencesCopyAppValue(checkInterval, app);
   if (!plist) {
-    const float fiveHoursInSeconds = 5.0 * 60.0 * 60.0;
+    const float fiveHoursInSeconds = 0.0;
     NSNumber* value = [NSNumber numberWithFloat:fiveHoursInSeconds];
     CFPreferencesSetAppValue(checkInterval, value, app);
     CFPreferencesAppSynchronize(app);
-- 
2.4.3

